packages/test/utils/mocks.ts

```diff
@@ -0,0 +1,24 @@
+import { axios } from '@baseapp-frontend/utils'
+
+import MockAdapter from 'axios-mock-adapter'
+import Cookies from 'js-cookie'
+
+jest.mock('js-cookie')
+
+export type CookiesGetByNameFn = (name: string) => string
+
+export const cookiesMock = jest.mocked(Cookies, true)
+// @ts-ignore TODO: investigate AxiosRequestHeaders error
```

Comment:
like RR said on the other PR, lets add the TODOS so some story?

Reply:
good catch, created this one for all TS errors https://app.approvd.io/projects/BA/stories/25108